src/arrayMethodJoin.js

```diff
@@ -4,8 +4,28 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    if (this.length) {
+      let joinedString = '';
+
+      for (let i = 0; i < this.length; i++) {
+        let buff = this[i];
+
+        if (this[i] == null) {
+          buff = '';
+        }
+
+        if (i === this.length - 1) {
+          joinedString += buff;
+
+          return joinedString;
+        }
+
+        joinedString += buff + `${separator}`;
```

Comment:
```suggestion
        joinedString += buff + separator;
```
Можна без інтерполяції